README.md

```diff
@@ -1 +1,63 @@
-# terraform-provider-mailgun
\ No newline at end of file
+Terraform Provider
```

Comment:
Can you please keep `Mailgun` in the title?

Reply:
It's done.